airflow/sensors/base_sensor_operator.py

```diff
@@ -111,9 +111,13 @@ def execute(self, context):
                 # This gives the ability to set up non-blocking AND soft-fail sensors.
                 if self.soft_fail and not context['ti'].is_eligible_to_retry():
                     self._do_skip_downstream_tasks(context)
-                    raise AirflowSkipException('Snap. Time is OUT.')
+                    raise AirflowSkipException(
+                        "Snap. Time is OUT for dag: {dag}"
+                        .format(dag=self.dag.dag_id if self.dag else ""))
```

Comment:
It should be impossible to run a task without a dag, so the conditional you have here isn't needed. (Or _shouldn't_ be.

Reply:
updated!